src/main/java/appeng/core/features/AEFeature.java

```diff
@@ -122,6 +122,7 @@ public boolean isVisible()
 	CraftingLog( Constants.CATEGORY_MISC, false ),
 	LightDetector( Constants.CATEGORY_MISC ),
 	DebugLogging( Constants.CATEGORY_MISC, false ),
+	ApiConflictCheck( Constants.CATEGORY_MISC, true ),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00115)